freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_replica_install.sh

```diff
@@ -44,6 +54,7 @@ ipa-replica-install \
           --mkhomedir \
           --ip-address "$IPADDR" \
           $FORWARDERS \
+          --auto-forwarders \
```

Comment:
I see this as a problem.

Scenario:

Initial config:
> ipaserver0 (primary) = 10.0.0.1 with forwarders 169.254.0.1 
ipaserver1 (replica) = 10.0.0.2 with forwarders 169.254.0.1 & 10.0.0.1 

Repair ipaserver0:
> ipaserver1 (replica) = 10.0.0.2 with forwarders 169.254.0.1 & 10.0.0.1 
ipaserver2 (replica promoted to new primary) = 10.0.0.3 with forwarders 169.254.0.1 & 10.0.0.2 

A new cluster is provisioned and 10.0.0.1 is allocated to a new FreeIPA cluster. DNS query goes to 10.0.0.3, it gets forwarded to 10.0.0.2, it gets forwarded to 10.0.0.1, and 10.0.0.1 responds with information from a cluster that is supposed to be isolated from this cluster.

1. Is there anything that can be done to remove the forwarder after `ipa-replica-install`?
2. If not, could we at least limit `--auto-forwarders` to only being used for the "192.254.x.x" case above?

Reply:
in theory `ipa dnsconfig-mod` could be used to modify forwarders, but I never tested

Reply:
Thanks for the feedback. We could do option 2. "least limit --auto-forwarders to only being used for the "169.x.x" case above"